waterloop-site/src/pages/SustainableTechEvent.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 
 import { AboutEvent } from '../components/SustainableTech/AboutEvent';
+import { Teams } from '../components/SustainableTech/Teams';
```

Comment:
#nitpick can you put this after the `Sponsors` to maintain alphabetical ordering?